src/components/form/user/User.tsx

```diff
@@ -192,7 +202,7 @@ const UserForm: React.FC = () => {
 
         <AddressModal
           isOpen={isAddAddressModalOpen}
-          onClose={() => setIsAddAddressModalOpen(false)}
+          onClose={() => closeAddAddressModal()}
```

Comment:
Maybe it's better this way. 
```suggestion
          onClose={closeAddAddressModal}
```